src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const {
+            id,
+            title,
+          } = tab;
+
+          return (
+            <li
+              key={id}
+              className={cn({ 'is-active': selectedTab.title === tab.title })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${id}`}
+                data-cy="TabLink"
+                onClick={() => selectedTab === tab || onTabSelected(tab)}
```

Comment:
You can write a function handler for it, not use anonim function ( not critical)